pact-lang-api.js

```diff
@@ -81,6 +81,34 @@ var toTweetNaclSecretKey = function(keyPair) {
   return hexToBin(keyPair.secretKey + keyPair.publicKey);
 };
 
+/**
+ * Attach signature to hashed data
+ * @param msg - some data to be passed to blake2b256.
+ * @param keyPair - signing ED25519 keypair
+ * @return {Array} of "hash", "sig" (signature in hex format), and "pubKey" public key values.
+ */
+var attachSig = function(msg, kpArray) {
+  var hshBin = hashBin(msg);
+  var hsh = base64UrlEncode(hshBin);
+  if (kpArray.length === 0 ) {
+   return [{hash: hsh, sig: undefined}];
+  } else {
+    return kpArray.map(kp => {
+      if (
+        (kp.hasOwnProperty("publicKey") && kp.publicKey) &&
+        (kp.hasOwnProperty("secretKey") && kp.secretKey)) {
+        return sign(msg, kp);
+      } else {
+        return {
+          hash: hsh,
+          sig: undefined,
```

Comment:
Should this be `sig` or `sigs`?

Reply:
This is `sig`

> Ooh nice, getting rid of the `.min.js` files definitely cleans things up. In a follow-on commit we should probably add `*.min.js` to the `.gitignore` file so they won't get added back accidentally in the future.

Applied the comments.